PSOperationsTests/PSOperationsTests.swift

```diff
@@ -973,4 +1006,54 @@ class PSOperationsTests: XCTestCase {
         XCTAssertEqual(1, opQ.operationCount)
         XCTAssertTrue(op.waitCalled)
     }
+    
+    /*
+        In 9.1 (at least) we found that occasionaly OperationQueue would get stuck on an operation
+        The operation would be ready, not finished, not cancelled, and have no dependencies. The queue
+        would have no other operations, but the op still would not execute. We determined a few problems
+        that could cause this issue to occur. This test was used to invoke the problem repeatedly. While we've
+        seen the opCount surpass 100,000 easily we figured 25_000 operations executing one right after the other was
+        a sufficient test and is still probably beyond typical use cases. We wish it could be more concrete, but it is not.
+    */
+    func testOperationQueueNotGettingStuck() {
+        
+        var opCount = 0
+        var requiredToPassCount = 25_000
+        let q = OperationQueue()
+        
+        let exp = expectationWithDescription("requiredToPassCount")
+        
+        func go() {
+            
+            if opCount >= requiredToPassCount {
+                exp.fulfill()
+                return
+            }
+            
+            let blockOp = BlockOperation {
+                (finishBlock: Void -> Void) in
+                finishBlock()
+                go()
```

Comment:
My concern is not the length of the test, but the probability of a missed KVO going up (presumably) when there's less and less time between op state changes (no create steps to get in the way). But if you've seen it more when adding an op, I wouldn't worry much about it.